Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1077,13 +1077,14 @@ class BrowserViewController: UIViewController,
 
         addChild(searchController)
         view.addSubview(searchController.view)
-        searchController.view.snp.makeConstraints { make in
-            make.top.equalTo(header.snp.bottom)
-            make.left.right.equalTo(view)
 
-            let constraintTarget = isBottomSearchBar ? overKeyboardContainer.snp.top : view.snp.bottom
-            make.bottom.equalTo(constraintTarget)
-        }
+        NSLayoutConstraint.activate([
+            searchController.view.topAnchor.constraint(equalTo: header.bottomAnchor),
+            searchController.view.leftAnchor.constraint(equalTo: view.leftAnchor),
+            searchController.view.rightAnchor.constraint(equalTo: view.rightAnchor)
+        ])
+        searchController.view.bottomAnchor.constraint(equalTo: overKeyboardContainer.topAnchor).isActive = isBottomSearchBar
+        searchController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = !isBottomSearchBar
```

Comment:
@lmarceau , I can see that line 1086 and 1087 is where the issue is coming from. What I'm trying to figure out is why the search engine view disappears when I remove the previous line 1080 `searchController.view.snp.makeConstraints { make in }`. But with this line of code and the new constraints, the search engines show up.